app/controllers/admin/edition_images_controller.rb

```diff
@@ -26,10 +26,13 @@ def create
     @new_image = @edition.images.build
     @new_image.build_image_data(image_params["image_data"])
 
+    @new_image.image_data&.validate_on_edition = @edition
+    @new_image.image_data&.validate_on_image = @new_image
```

Comment:
It seems unusual to have the safe operator on an assignment. I'd imagine the code would be more readable putting it inside a conditional block

i.e. 

```
if @new_image.image_data
   @new_image.image_data.validate_on_edition = @edition
   @new_image.image_data.validate_on_image = @new_image
end
```